MmSupervisorPkg/Core/PrivilegeMgmt/SyscallDispatcher.c

```diff
@@ -238,23 +238,25 @@ SyscallDispatcher (
     mPcdCheck     = FALSE;
   }
 
-  while (!AcquireSpinLockOrFail (mCpuToken)) {
-    CpuPause ();
-  }
+  if (FeaturePcdGet (PcdEnableSyscallLogs)) {
+    while (!AcquireSpinLockOrFail (mCpuToken)) {
+      CpuPause ();
+    }
 
-  DEBUG ((
-    DEBUG_VERBOSE,
-    "%a Enter... CallIndex: %lx, Arg1: %lx, Arg2: %lx, Arg3: %lx, CallerAddr: %p, Ring3Stack %p\n",
-    __FUNCTION__,
-    CallIndex,
-    Arg1,
-    Arg2,
-    Arg3,
-    CallerAddr,
-    Ring3StackPointer
-    ));
+    DEBUG ((
+      DEBUG_VERBOSE,

```

Comment:
Given the message is opted into now with the PCD, do you think this should be `DEBUG_INFO`?

That would print the message if either info or verbose level is enabled + PCD is true as opposed to just verbose + PCD is true.

Reply:
Yeah, I was thinking the same thing :) Will update shortly.